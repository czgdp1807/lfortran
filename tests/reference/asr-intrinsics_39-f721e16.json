{
    "basename": "asr-intrinsics_39-f721e16",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_39.f90",
    "infile_hash": "bfb285bb5819a2fc6c53b75b7de2c7f3dca58181039b9d968af4abfd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_39-f721e16.stdout",
    "stdout_hash": "b6c46418bd226ce4b69383786114932f0a74cc909fa8f3ed91cebfff",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}