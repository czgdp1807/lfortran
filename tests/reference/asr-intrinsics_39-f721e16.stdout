(TranslationUnit
    (SymbolTable
        1
        {
            test_time_and_date:
                (Program
                    (SymbolTable
                        2
                        {
                            date:
                                (Variable
                                    2
                                    date
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Character 1 8 () PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            time:
                                (Variable
                                    2
                                    time
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Character 1 10 () PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            values:
                                (Variable
                                    2
                                    values
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 8 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            zone:
                                (Variable
                                    2
                                    zone
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Character 1 5 () PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    test_time_and_date
                    []
                    [(IntrinsicImpureSubroutine
                        DateAndTime
                        [(Var 2 date)
                        (Var 2 time)
                        (Var 2 zone)
                        (Var 2 values)]
                        0
                    )
                    (Print
                        (StringFormat
                            (StringConstant
                                "(a,2x,a,2x,a)"
                                (Character 1 13 () PointerString)
                            )
                            [(Var 2 date)
                            (Var 2 time)
                            (Var 2 zone)]
                            FormatFortran
                            (Character -1 0 () PointerString)
                            ()
                        )
                    )
                    (Print
                        (StringFormat
                            (StringConstant
                                "(8i5)"
                                (Character 1 5 () PointerString)
                            )
                            [(Var 2 values)]
                            FormatFortran
                            (Character -1 0 () PointerString)
                            ()
                        )
                    )]
                )
        })
    []
)
